syntax = "proto3";
package easy_work_service;

option go_package = "go.easyops.local/contracts/protorepo-models/easyops/model/easy_work_service";





/*
应用系统
*/
message AppSystem {
    /*
    应用系统 ID
    */
    string instanceId = 1 ;
    /*
    名称
    */
    string name = 2 ;
    /*
    链接地址
    */
    string url = 3 ;
    /*
    图标
    */
    string icon = 4 ;
    /*
    类型
    */
    string category = 5 ;
    /*
    描述
    */
    string memo = 6 ;
}

